### ISSUE TITLE
Bio should open immediately, no button

### ISSUE THREAD

**Issue by benaltair** [bahaistudies/jbs#37]
This plus should just open a biography, rather than making it two steps.

I get that it's because you can enter a one-line affiliation thing, but it makes more sense to just include that in the bio modal.

*Source url:* https://journal.bahaistudies.ca/online/article/view/147
*Reported by:* 
*Reported at:* 09 Mar at 04:53 UTC

*Browser:* Chrome 88.0.4324.190
*User Agent:* Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.190 Safari/537.36
*OS:* Windows 10.0
*Screen:* 1920x1080
*Viewport:* 1920x947
*Referrer:* https://github.com/
*Screenshot:*
![Screenshot](https://ybug.io/data/reports/r1sbw9hq0kkhj9nxf2yt/vsrk2tgf0pca0t3jg4kv/screenshot.jpg?_token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE2MTUyNjU2NTMsImRhdGEiOnsicmVwb3J0IjoidnNyazJ0Z2YwcGNhMHQzamc0a3YiLCJ1c2VyIjoiR2l0SHViIn19.jGRTKzOGXBqmrI_DQrk5poLfnPPeKeYmhVL2eLkrf14)

For more details please visit report page on Ybug:
[https://ybug.io/dashboard/reports/detail/vsrk2tgf0pca0t3jg4kv](https://ybug.io/dashboard/reports/detail/vsrk2tgf0pca0t3jg4kv)

**Comment by benaltair** [comment]
Can be done at the same time as #33 perhaps.

**Comment by benaltair** [comment]
The less steps the better. 

- [ ] Modal should open immediately when clicking the name, no button
- [ ] `aria-label` needs to be added to each element, describing what will happen when clicking link

![image](https://user-images.githubusercontent.com/17771229/116310624-1e7a5f00-a75f-11eb-9056-ef0d2e1e25c2.png)

For the second item, it is not currently clear what happens when clicking on the authors name (for anyone for that matter). 
```html
<ul class="authors-string" aria-label="Authors">
     <li><a class="author-string-href" href="#author-1"><span>Dr. Todd Smith</span><sup class="author-symbol author-plus">+</sup><sup class="author-symbol author-minus hide">−</sup></a></li>
</ul>
```

**Comment by benaltair** [comment]
Structuring the list like this sounds good so far in screen reader testing. `aria-label` adds context for the list, `aria-hidden` stops the decorative `+` from being read out, and a `title` attribute was used instead of `aria-label` since it gives additional information for what the link does (auxiliary to the content of the link). If `aria-label` is used here, it would override the link contents on most screen readers, requiring redundant text like the author's name.
```html
<ul class="authors-string" aria-label="Authors">
  <li>
    <a class="author-string-href" href="#author-1" title="Open Bio">
      <span>Dr. Todd Smith</span><sup class="author-symbol author-plus" aria-hidden="true">+</sup>
    </a>
  </li>
</ul>
```
This should be tested on longer lists of authors to make sure it flows nicely.
